lib/middleware/jsonIDValidator.ts

```diff
@@ -15,7 +15,7 @@ export type ValidateOptions = {
 export const uniqueIDValidator = (options?: ValidateOptions): MiddlewareRequest => {
   return async (req: NextApiRequest, res: NextApiResponse): Promise<MiddlewareReturn> => {
     try {
-      if (options?.runValidationIf && options.runValidationIf(req) === false) {
+      if (options?.runValidationIf?.(req) === false) {
```

Comment:
Funny that it does work for you. I tried this approach but I was always getting an error from VSCode.